modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -104,33 +205,102 @@ The `content` argument can no longer be used with `mceInsertClipboardContent`. I
+
+Three additional options are now supported when Real-time Collaboration is active:
+
+* `valid_elements`
+* `valid_children`
+* `extended_valid_elements`
+
+Real-time Collaboration now supports xref:autocompleter.adoc[the `autocompleter` API].
+
+Real-time Collaboration now supports xref:content-behavior-options.adoc#inline_boundaries[the `inline_boundaries` option].
+
+When provided values are incorrect, the returned error messages are now less opaque.
+
+For example, the error message `'key' was not a string` is now the error message `Expected field 'key'`.
+
+Previously, if a {productname} editor instance with Real-time Collaboration loaded also contained a Horizontal Rule (ie an `+<hr>+` element), attempting to move the insertion point over the Horizontal Rule using arrow keys threw an exception: `Uncaught Error: Unsupported node type: [object DocumentType]`.
+
+With this update, moving the insertion point through a Horizontal Rule using arrow keys works as expected: the insertion point moves and no exception is thrown.
+
+Previously, attempting to remove failed image uploads with the `images_upload_handler` API failed if Real-time Collaboration was loaded.
+
+With this update, `images_upload_handler` removes failed image uploads and presents the correct alert message.
+
+Keyboard events can now be prevented before the Real-time Collaboration plugin intercepts them.
+
+Previously, selecting linked images when the `link_context_toolbar` option was enabled, caused the Real-time Collaboration editor to enter an endless loop. This loop caused the selected images to flicker.
+
+With this update, the Real-time Collaboration editor no longer enters an endless loop when linked images are selected. Consequenlty, the selected images no longer flicker.
+
+
+=== Spell Checker Pro
+
+The free `spellchecker` plugin was removed as no longer supported.
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro], which offers equivalent functionality and more, is available.
+
+The following fixes were made to the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin:
+
+Previously, Spell Checker Pro treated soft hyphens (ie the `+&shy;+` html entity) as a word boundary. As a consequence words containing soft hyphens were not correctly spellchecked as whole words.
+
+With this update, `+&shy;+` entity is correctly recognised as part of a word, not a word boundary. And words containing soft hyphens are, consequently, spell checked correctly.
+
+As well, one command and one option have been replaced and a, previously deprecated, fallback option has been removed.
+
+[cols="1,1,1"]
+|===
+| Removed item                    | Replacement           | Element type
+
+| `mceSpellcheckDocument`         | `mceSpellcheckDialog` | Command
+
+| `spellchecker_select_languages` | `content_langs`       | Option
+
+| `spellchecker_whitelist`        | No replacement        | Option
```

Comment:
This is incorrect, the replacement is `spellchecker_ignore_list` (or at least it's the same scenario as `content_langs`).

Reply:
Corrected.